README.md

```diff
@@ -248,7 +248,7 @@ This contract can be used in order to browse and get a feel of the COTI network.
 1.  Navigate to the `examples` directory in the `confidentiality-contracts` directory inside the `coti-sdk-python-examples` project.
 
     ```bash
-    cd confidentiality-contracts/contracts/examples
+    cd confidentiality-contracts/contracts
```

Comment:
but you didnt navigate to the examples folder here...

Reply:
I will change it to read "navigate to the contracts directory"